### DISCUSSION TITLE
Youtube-BB cropped data

### DISCUSSION THREAD

**Discussion by meetdave06** [STVIR/pysot#222] [General]
It is mentioned in the [training readme](https://github.com/STVIR/pysot/blob/master/TRAIN.md#prepare-training-dataset) file that the old link for cropped data is wrong for Youtube BB dataset. Does this mean that the steps listed in the [training_dataset](https://github.com/STVIR/pysot/tree/master/training_dataset/yt_bb) directory to generate crops for Youtube BB Dataset produces incorrect crops?
As I am in US, I don't have access to the cropped dataset in the new link provided as it is hosted on Baidu net. Thanks.

**Comment by 121649982** [comment]
I　can download in china.but i don't know how to use it.

**Comment by haishibei** [comment]
@121649982 ，sorry to bother you, Now i can't download the dateset "youtubebb",can you share it with me, My wechat is "jzb12390", thank you sincerely！
